### PR TITLE
Ce glider

### PR DESCRIPTION
**Pull request by s-pearce**
Adds CE glider deployment 386-12 and recoveries 312-10 & 311-10 and associated cruise and calibrations.
Also fixes a type that was in a PAR instrument scaling factor potentially causing erroneous values.

### CODE REVIEW

**Comment by s-pearce** [comment]
@dbuffitt Sorry Derrick, I must have accidentally clicked your name for a review.  I have removed it now, but I am not sure if Github still requires you to review or not.
@cwingard @desiderr can one of you review and merge?  Thanks.

**Comment by dbuffitt** [comment]
No problem, looks like I’m removed and do not need to review…

File: calibration/PARADM/CGINS-PARADM-50164__20130304.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 serial,name,value,notes
-50164,CC_bsipar_par_scaling,1.00E-05,
+50164,CC_bsipar_par_scaling,1.00E-06,
```
Comment (cwingard) [inline calibration/PARADM/CGINS-PARADM-50164__20130304.csv:2]: If we are changing this calibration value, then I need to update the Metadata Change Log. Are there more of these?

File: calibration/FLORTM/CGINS-FLORTM-03186__20191025.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
```
Comment (cwingard) [inline calibration/FLORTM/CGINS-FLORTM-03186__20191025.csv:1]: Where is this cal sheets on doris so I can double check?

**Review by desiderr — CHANGES_REQUESTED** [review]
2 issues:

Did you recover Glider 311 at 2:47AM local? This might be a typo in the cruise report.
Glider 386 is listed as being deployed on **2019**-09-17 instead of 2020.

File: calibration/PARADM/CGINS-PARADM-50164__20130304.csv (hunk 1 above)
Comment (s-pearce) [reply calibration/PARADM/CGINS-PARADM-50164__20130304.csv:2]: No, just the one.  I grepped the PARADM directory and that looked to be the only one with 1e-5.
Also I don't know anything about Metadata Change Logs.  Lets chat about that.

File: calibration/FLORTM/CGINS-FLORTM-03186__20191025.csv (hunk 2 above)
Comment (s-pearce) [reply calibration/FLORTM/CGINS-FLORTM-03186__20191025.csv:1]: doris gliders:data/386/000012/8304-61003-386-00012_Configuration_Record.xlsx

**Comment by s-pearce** [comment]
@desiderr No those are not typos.  We deployed 386 around Noon PDT on the 17th and then a 13 hour transit to 311 where we recovered about 02:30 PDT on the 18th and then another long transit to recover 312 later that day.

**Comment by desiderr** [comment]
OK - but how about Glider 386? This last deployment is #12 - this was deployed a week or two ago in 2020, not in 2019, yes?

**Comment by s-pearce** [comment]
@desiderr sorry, I was in a hurry while reading and answering your last question.  You are correct, 2019 IS a typo.  It should be 2020.  I have fixed and pushed the new commit to the pull request.

**Comment by desiderr** [comment]
OK, I'm satisfied, I also checked serial numbers.

**Review by desiderr — APPROVED** [review]

**Review by cwingard — APPROVED** [review]

**Pull request merged**